ISSUE_TEMPLATE

```diff
@@ -0,0 +1,17 @@
+### Context
+
+* [ ] **mapbox-gl-js version**: 
```

Comment:
Instead of asking for the mapbox-gl-js version they're using, can we insist that they use the latest version?